source/extensions/listener_managers/listener_manager/filter_chain_manager_impl.h

```diff
@@ -352,5 +353,10 @@ class FilterChainManagerImpl : public Network::FilterChainManager,
   // Index filter chains by name, used by the matcher actions.
   FilterChainsByName filter_chains_by_name_;
 };
+
+namespace FilterChain {
```

Comment:
is this to avoid some naming conflicts?

Reply:
I added this namespace name because I don't think it's possible to refer to anonymous namespaces externally, which is needed to call the `forceRegister...` function. Open to better names.